### ISSUE TITLE
Update code to use `{{this.property}}` syntax where appropriate

### ISSUE THREAD

**Issue by locks** [ember-learn/super-rentals#112]
In an effort to clarify where a value in a template interpolation comes from (`{{}}`), we started prefixing local properties (that belong to the Controller/Component of the respective template) with `this`, like the following:

```javascript
import Component from '@ember/component';

export default Component.extend({
  framework: 'Ember.js'
});
```

```hbs
{{this.framework}}
```

This repository needs to be updated to match v3.6 of the Guides.

**Comment by Windvis** [comment]
I can take a look at this tonight.

**Comment by jenweber** [comment]
closed with #113 

Thank you!!! 🎉
